functions/src/index.ts

```diff
@@ -146,123 +106,20 @@ export const beforecreated: BlockingFunction = beforeUserCreated(
-        { date: new Date('2024-01-31'), value: 114 },
-        { date: new Date('2024-01-30'), value: 123 },
-        { date: new Date('2024-01-29'), value: 109 },
-        { date: new Date('2024-01-28'), value: 105 },
-        { date: new Date('2024-01-27'), value: 98 },
-        { date: new Date('2024-01-26'), value: 94 },
-        { date: new Date('2024-01-25'), value: 104 },
-        { date: new Date('2024-01-24'), value: 102 },
-      ],
-      diastolicBloodPressure: [
-        { date: new Date('2024-02-01'), value: 70 },
-        { date: new Date('2024-01-31'), value: 82 },
-        { date: new Date('2024-01-30'), value: 75 },
-        { date: new Date('2024-01-29'), value: 77 },
-        { date: new Date('2024-01-28'), value: 72 },
-        { date: new Date('2024-01-27'), value: 68 },
-        { date: new Date('2024-01-26'), value: 65 },
-        { date: new Date('2024-01-25'), value: 72 },
-        { date: new Date('2024-01-24'), value: 80 },
-      ],
-      heartRate: [
-        { date: new Date('2024-02-01'), value: 79 },
-        { date: new Date('2024-01-31'), value: 62 },
-        { date: new Date('2024-01-30'), value: 77 },
-        { date: new Date('2024-01-29'), value: 63 },
-        { date: new Date('2024-01-28'), value: 61 },
-        { date: new Date('2024-01-27'), value: 70 },
-        { date: new Date('2024-01-26'), value: 67 },
-        { date: new Date('2024-01-25'), value: 80 },
-        { date: new Date('2024-01-24'), value: 65 },
-      ],
-      weight: [
-        { date: new Date('2024-02-01'), value: 269 },
-        { date: new Date('2024-01-31'), value: 267 },
-        { date: new Date('2024-01-30'), value: 267 },
-        { date: new Date('2024-01-29'), value: 265 },
-        { date: new Date('2024-01-28'), value: 268 },
-        { date: new Date('2024-01-27'), value: 268 },
-        { date: new Date('2024-01-26'), value: 266 },
-        { date: new Date('2024-01-25'), value: 266 },
-        { date: new Date('2024-01-24'), value: 267 },
-      ],
-      dryWeight: 267,
-    },
-    symptomScores: [
-      {
-        overall: 40,
-        physicalLimits: 50,
-        socialLimits: 38,
-        qualityOfLife: 20,
-        specificSymptoms: 60,
-        dizziness: 50,
-        date: new Date('2024-01-24'),
-      },
-      {
-        overall: 60,
-        physicalLimits: 58,
-        socialLimits: 75,
-        qualityOfLife: 37,
-        specificSymptoms: 72,
-        dizziness: 70,
-        date: new Date('2024-01-15'),
-      },
-      {
-        overall: 44,
-        physicalLimits: 50,
-        socialLimits: 41,
-        qualityOfLife: 25,
-        specificSymptoms: 60,
-        dizziness: 50,
-        date: new Date('2023-12-30'),
-      },
-      {
-        overall: 75,
-        physicalLimits: 58,
-        socialLimits: 75,
-        qualityOfLife: 60,
-        specificSymptoms: 80,
-        dizziness: 100,
-        date: new Date('2023-12-15'),
-      },
-    ],
-  })
+interface HealthSummaryInput {
```

Comment:
Might be great to use this mechanism in other cloud functions such as the invitation code one as well; nice mechanism.